SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -1,22 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-<!--
+  <!--
```

Comment:
There are quite a few formatting changes to this file, I will point out the bug fix.